jumpscale/packages/owncloud/package.py

```diff
@@ -0,0 +1,25 @@
+import toml
+from pathlib import Path
+
+PACKAGE_CONFIG_PATH = Path(__file__).parent.resolve() / 'package.toml'
+class owncloud:
+    def install(self, **kwargs):
+        domain = kwargs.get('domain')
+        letsencryptemail = kwargs.get('letsencryptemail')
+        if not all([domain, letsencryptemail]):
+            raise(f'usage: j.servers.threebot.default.packages.add(path="<package_path>", domain="<domain>", letsencryptemail="<letsencryptemail>")')
+        
+        with open(PACKAGE_CONFIG_PATH, 'rt') as f:
+            parsed_toml = toml.load(f)
+        
+        if domain:
+            parsed_toml['servers'][0]['domain'] = domain
+            print(f"set the domain to {domain}")
```

Comment:
can we use log instead of print if possible?

Reply:
done.